agent/agent.go

```diff
@@ -3,6 +3,8 @@ package agent
 import (
 	"context"
 	"fmt"
+	"github.com/influxdata/telegraf/plugins/inputs"
```

Comment:
Make sure you follow golang convention.
See "Formatting Imports" on https://www.digitalocean.com/community/tutorials/importing-packages-in-go
Golang has a lot of tools that do formatting and linting and things like that for you, such as `govet`, `golint` and `goimports`.